src/Telemetry/AzureDevOpsTimeline/AzureDevOpsTimeline.cs

```diff
@@ -319,5 +364,56 @@ private static string GetBucket(AugmentedTimelineIssue augIssue)
         {
             return await azureServer.ListBuilds(project, cancellationToken, minDateTime, limit);
         }
+
+        private async Task GetImageNames(List<AugmentedTimelineRecord> records, CancellationToken cancellationToken)
+        {
+            SemaphoreSlim throttleSemaphore = new SemaphoreSlim(50);
+
+            var bag = new ConcurrentBag<Task>();
+            int canceledTasksNumber = 0;
```

Comment:
```suggestion
            int cancelledTaskCount = 0;
```
How about this? (there's LL typo too)

Reply:
looks good